runtime/dancebox/tests/common/xcm/constants.rs

```diff
@@ -204,7 +196,11 @@ pub mod westend {
             ..Default::default()
         };
 
-        genesis_config.build_storage().unwrap()
+        ChainSpec::builder(westend_runtime::WASM_BINARY.unwrap(), None)
+            .with_genesis_config(serde_json::to_value(&genesis_config).unwrap())
+            .build()
+            .build_storage()
+            .unwrap()
```

Comment:
I believe this change makes CI take far longer for this test than it should because each test creates compiles the wasm once. Not sure if we can improve it however.